projects/batfish-client/src/main/java/org/batfish/client/Client.java

```diff
@@ -1501,15 +1501,31 @@ private boolean help(List<String> options, List<String> parameters) {
       return true;
    }
 
-   private boolean initContainer(String[] words) {
-      if (words.length > 2) {
-         _logger.errorf("Invalid arguments: %s\n", Arrays.toString(words));
-         printUsage(Command.INIT_CONTAINER);
+   private boolean initContainer(List<String> options, List<String> parameters) {
+      if (!isValidArgument(options, parameters, 1, 0, 1, Command.INIT_CONTAINER)) {
          return false;
       }
-      String containerPrefix = (words.length > 1) ? words[1]
-            : DEFAULT_CONTAINER_PREFIX;
-      _currContainerName = _workHelper.initContainer(null, containerPrefix);
+      if (!options.isEmpty()) {
```

Comment:
It's okay, but I bet you could simplify this code a lot by ordering your `if`s better and unnesting. The current code is way more complex than it needs to be IMO ; hard to understand.

Example:

```java
if (options.contains("-setname")) {
   validate using =1 option, using =1 parameter
   _currContainerName = _workHelper.initCountainer(parameters.get(0), null));
}
else {
   validate using =0 options, 0 or 1 parameter.
   existing 2 lines of logic
}
final 3 lines of logs.
```